roles/fluentd_node/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+# TODO: Update fluentd install and configuration when packaging is complete
+- name: download and install td-agent
+  yum:
+    name: 'http://packages.treasuredata.com/2/redhat/7/x86_64/td-agent-2.2.0-0.x86_64.rpm'
+    state: present
+
+- name: install Kubernetes fluentd plugin
+  command: '/opt/td-agent/embedded/bin/gem install fluent-plugin-kubernetes'
```

Comment:
same as above, I don't like this, but I understand this will be fixed once we have it packaged.

But it'd be best to not have to run this every single time we config.